modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm

```diff
@@ -77,41 +77,42 @@
 			switch(rand(1,6))
 				if(1)
 					if(stress >= 100)
-						to_chat(owner, "<font color=purple>You feel slightly aroused...</font>")
+						to_chat(owner, "<font color=purple>Your breath grows hot and heavy...</font>")
+						owner.adjustOxyLoss(8)
+						owner.emote("exhale")
 					else
-						to_chat(owner, "<font color=purple>Lust spreads over your body!</font>")
-						owner.emote("moan")
+						to_chat(owner, "<font color=purple>You feel hot and seduced!</font>")
+						owner.adjustOxyLoss(16)
+						owner.emote("gasp")
+						
 				if(2)
 					if(stress >= 100)
 						to_chat(owner, "<font color=purple>You can't stop shaking...</font>")
 						owner.do_jitter_animation(20)
+						owner.emote("shiver")
 					else
-						to_chat(owner, "<font color=purple>You feel hot and seduced!</font>")
+						to_chat(owner, "<font color=purple>Lust spreads over your body!</font>")
 						owner.dizziness += 20
 						owner.add_confusion(20)
 						owner.Jitter(20)
 						owner.do_jitter_animation(20)
-						owner.adjustStaminaLoss(50)
+						owner.emote("twitch")
 				if(3, 4)
 					if(stress >= 100)
-						to_chat(owner, "<font color=purple>You bring your hips together in lust.</font>")
+						to_chat(owner, "<font color=purple>You bring your hips together in lust....</font>")
```

Comment:
That's one too many dots there.

Reply:
oh my GOD i cannot believe I've done this